test/swim_test.js

```diff
@@ -191,14 +192,14 @@ test('suspicion subprotocol cannot be reenabled without all timers first being s
 });
 
 test('marks member faulty after suspect period', function t(assert) {
-    assert.plan(2);
+    assert.plan(1);
 
     var suspicion = createSuspicion();
     var member = suspicion.ringpop.membership.remoteMember;
     suspicion.setTimeout = function(fn) { return fn(); };
-    suspicion.ringpop.membership.update = function(changes) {
-        assert.equals(changes[0].address, member.address, 'updates correct member');
-        assert.equals(changes[0].status, 'faulty', 'marks member faulty');
+    // TODO Brittle test, just use real membership object
```

Comment:
yes, i dont like it either. was a bad decision made by me. i have to remove these kinds of tests when time allows